docs/tsg_collect_logs.md

```diff
@@ -0,0 +1,68 @@
+---
+id: tsgcollectlogs
+title: Collecting Logs
+sidebar_label: Collecting Logs
+---
+
+## Overview of OpenEBS Logger 
+
+Logger is a Kubernetes job which can be run on a cluster to extract pod logs and cluster information. It helps in troubleshoot/debugging activities. Logger runs the logger container *openebs/logger* and it is recommended to run for a specific duration to capture logs while attempting to reproduce issues.
+
+### Benefits
+
+Following are the benefits of using Logger.
+
+- The purpose of Logger is to obtain debug-information/quick logs in clusters where more standard logging frameworks like EFK are not already configured. Logger creates a simple support bundle which can be provided to debug teams.
+- This may be the case with most "non-production/development" infrastructures.
+
```

Comment:
Let us add "Where is the logger found" - and point to the location where the kubernetes job is available (in openebs/test-storage)

Reply:
Done